examples/flutter_gallery/lib/gallery/options.dart

```diff
@@ -391,9 +392,9 @@ class GalleryOptionsPage extends StatelessWidget {
   List<Widget> _enabledDiagnosticItems() {
     // Boolean showFoo options with a value of null: don't display
     // the showFoo option at all.
-    if (null == options.showOffscreenLayersCheckerboard
-             ?? options.showRasterCacheImagesCheckerboard
-             ?? options.showPerformanceOverlay)
+    if (null == options.showOffscreenLayersCheckerboard ??
+        options.showRasterCacheImagesCheckerboard ??
+        options.showPerformanceOverlay)
```

Comment:
Isn't there a bug here? `==` is done before `??`. So the code is the same as `(null == options.showOffscreenLayersCheckerboard) ?? options.showRasterCacheImagesCheckerboard ?? options.showPerformanceOverlay`. 
I guess it should be:

```dart
if (null == (options.showOffscreenLayersCheckerboard ?? 
    options.showRasterCacheImagesCheckerboard ?? 
    options.showPerformanceOverlay)) {
...
```